tests/qa_automation/execute_test_run.pm

```diff
@@ -34,6 +34,10 @@ sub run {
 
     #output result to serial0 and upload test log
     if (get_var("QA_TESTSUITE")) {
+        #calm down from previous test in case
+        #  1)runfile contain fs_stress related test
+        #  2)previous run_log stream too much to finish writting into /tmp
+        sleep 2;
```

Comment:
why not call assert_script_run("sync") instead? 2 sounds too much like a magic number...